tests/dbgp/dbgpclient.php

```diff
@@ -60,8 +93,7 @@ function doRead( $conn )
 			$read = preg_replace( '@(2002-20[0-9]{2})@', '2002-2099', $read );
 			echo $read, "\n\n";
 
-			if ( preg_match( '@<stream xmlns="urn.debugger_protocol_v1" xmlns:xdebug@', $read ) )
-			{
+			if ( preg_match( '@<stream xmlns="urn.debugger_protocol_v1" xmlns:xdebug@', $read ) ) {
```

Comment:
Hmm - different standards I use for PHP scripts. Sorry. I'll make a PR to fix all of them. It would be nice if your patch did not change any existing CS though. It'd be cleaner to just fix it with a separate PR. Sorry :-/